peloton-to-garmin.py

```diff
@@ -107,12 +117,19 @@
     logger.info("Get workout summary")
     workoutSummary = api.getWorkoutSummaryById(workoutId)
 
-    logger.info("Writing TCX file")
     try:
-        tcx_builder.workoutSamplesToTCX(workout, workoutSummary, workoutSamples, output_directory)
+        title, filename, garmin_activity_type = tcx_builder.workoutSamplesToTCX(workout, workoutSummary, workoutSamples, output_directory)
+        logger.info("Writing TCX file: " + filename)
     except Exception as e:
-        logger.error("Failed to write TCX file for workout {} - Exception: {}".format(workoutId, e))
-    
+        logger.error("Failed to write TCX file for workout {} - {} - Exception: {}".format(workoutId, e))
+
+    if garmin_email is not None:
```

Comment:
TODO: after #19 is merged may update this so can be overidden by flag passed in on command line.  In case you ever want to generate TCX without uploading.